### ISSUE TITLE
Front page shortcuts

### ISSUE THREAD

**Issue by JosephKiranBabu** [freedombox/Plinth#1007]
I think we're a bit inconsistent on how we show front page shortcuts. It's currently like this:

1. Services which have web clients hosted on the FreedomBox get front page shortcuts.
2. Services which have non-web clients get front page shortcuts too (like privoxy and mumble).
3. Some services don't get front page shortcuts, like Tor, though it belongs in the same category of privoxy and mumble.

Tor browser on desktop and mobile can be considered a client, hence Tor can get shortcuts to clients in its service page as in #931 and also get a front page shortcut.
Tor is also a service to access FreedomBox (like Pagekite or Dynamic DNS) and can be moved to system as suggested by mray in #864 .

So, for consistency we can do one of two things:
1. Show front page shortcuts only for services that have web applications.
2. Show front page shortcuts for all services listed in the apps page.

**Comment by mray** [comment]
I agree we don't have a consistent approach here and I support your suggestion 1.
> 1. Show front page shortcuts only for services that have web applications.

 On top of the problem you describe there should be a distinction between anonymous/authenticated users. One should *also* be able to remove shortcuts for not logged in visitors.
